datafusion/core/src/datasource/physical_plan/parquet/statistics.rs

```diff
@@ -323,17 +324,6 @@ fn collect_scalars<I: Iterator<Item = Option<ScalarValue>>>(
     }
 }
 
-/// What type of statistics should be extracted?
-#[derive(Debug, Clone, Copy, PartialEq, Eq, PartialOrd, Ord, Hash)]
-pub enum RequestedStatistics {
-    /// Minimum Value
-    Min,
-    /// Maximum Value
-    Max,
-    /// Null Count, returned as a [`UInt64Array`])
-    NullCount,
-}
-
```

Comment:
I agree we do not need this. We store each min/max/row_count as an array instead :thus